prow/config/config.go

```diff
@@ -352,6 +362,9 @@ type Deck struct {
 	Branding *Branding `json:"branding,omitempty"`
 	// GoogleAnalytics, if specified, include a Google Analytics tracking code on each page.
 	GoogleAnalytics string `json:"google_analytics,omitempty"`
+	// RerunAuthConfig specifies who will be able to trigger job reruns when we allow this feature.
+	// Currently, this does nothing.
```

Comment:
Could we do this in the same PR? Fine either way

Reply:
I think the next changes are going to be tricky and I'm expecting a lot of discussion, so I'd rather keep them separate